lib/bifrost/topic.rb

```diff
@@ -63,8 +70,8 @@ def remove_subscriber(subscriber)
       end
     end
 
-    def ==(another_topic)
-      self.name == another_topic.name && self.class == another_topic.class
+    def ==(other)
```

Comment:
@lfportal for future reference please do not make changes outside the scope of the issue raised